pkg/conversion/mapi2capi/aws_fuzz_test.go

```diff
@@ -53,26 +53,52 @@ var _ = Describe("AWS Fuzz (mapi2capi)", func() {
 		},
 	}
 
-	fromMachineAndAWSMachineAndAWSCluster := func(machine *capiv1.Machine, infraMachine client.Object, infraCluster client.Object) capi2mapi.MachineAndInfrastructureMachine {
-		awsMachine, ok := infraMachine.(*capav1.AWSMachine)
-		Expect(ok).To(BeTrue(), "input infra machine should be of type %T, got %T", &capav1.AWSMachine{}, infraMachine)
-
-		awsCluster, ok := infraCluster.(*capav1.AWSCluster)
-		Expect(ok).To(BeTrue(), "input infra cluster should be of type %T, got %T", &capav1.AWSCluster{}, infraCluster)
-
-		return capi2mapi.FromMachineAndAWSMachineAndAWSCluster(machine, awsMachine, awsCluster)
-	}
-
-	conversiontest.MAPI2CAPIRoundTripFuzzTest(
-		scheme,
-		infra,
-		infraCluster,
-		mapi2capi.FromAWSMachineAndInfra,
-		fromMachineAndAWSMachineAndAWSCluster,
-		conversiontest.ObjectMetaFuzzerFuncs("openshift-machine-api"),
-		conversiontest.MAPIMachineFuzzerFuncs(&mapiv1.AWSMachineProviderConfig{}, awsProviderIDFuzzer),
-		awsProviderSpecFuzzerFuncs,
-	)
+	Context("AWSMachine Conversion", func() {
+		fromMachineAndAWSMachineAndAWSCluster := func(machine *capiv1.Machine, infraMachine client.Object, infraCluster client.Object) capi2mapi.MachineAndInfrastructureMachine {
+			awsMachine, ok := infraMachine.(*capav1.AWSMachine)
+			Expect(ok).To(BeTrue(), "input infra machine should be of type %T, got %T", &capav1.AWSMachine{}, infraMachine)
+
+			awsCluster, ok := infraCluster.(*capav1.AWSCluster)
+			Expect(ok).To(BeTrue(), "input infra cluster should be of type %T, got %T", &capav1.AWSCluster{}, infraCluster)
+
+			return capi2mapi.FromMachineAndAWSMachineAndAWSCluster(machine, awsMachine, awsCluster)
+		}
+
+		conversiontest.MAPI2CAPIMachineRoundTripFuzzTest(
+			scheme,
+			infra,
+			infraCluster,
+			mapi2capi.FromAWSMachineAndInfra,
+			fromMachineAndAWSMachineAndAWSCluster,
+			conversiontest.ObjectMetaFuzzerFuncs("openshift-machine-api"),
```

Comment:
Nit, could we set the namespace in a const so we avoid typing it every time?

Reply:
I have a follow up PR ready to open, can fix in that